recipes/mobie_utils/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.2.0" %}
+{% set name = "mobie_utils" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mobie/mobie-utils-python/archive/{{ version }}.tar.gz
+  sha256: 4af9ca43e439824ee15640a1391a662696b83f365a1ec6548d9321e90aff466b
+
+build:
+  number: 0
+  script: "pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  noarch: python
+  entry_points:
```

Comment:
Can you add tests under the `test`->`commands` section:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands